cli/azd/resources/apphost/templates/main.bicept

```diff
@@ -45,7 +45,13 @@ output SERVICE_BINDING_{{alphaSnakeUpper $name}}_ENDPOINT string = resources.out
 output SERVICE_BINDING_{{alphaSnakeUpper $name}}_CONNECTION_STRING string = resources.outputs.SERVICE_BINDING_{{alphaSnakeUpper $name}}_CONNECTION_STRING
 {{end -}}
 {{range $name, $value := .StorageAccounts -}}
-{{range $cname := $value.Containers -}}
+{{range $cname := $value.Blobs -}}
+output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = resources.outputs.SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT
+{{end -}}
+{{range $cname := $value.Tables -}}
+output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = resources.outputs.SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT
+{{end -}}
+{{range $cname := $value.Queues -}}
```

Comment:
Do these output names need another component so blob/table/queue do not conflict and generate the same names?

Reply:
I don't think so - the `$cname` value here is bound to the name inside the `Tables` slice on the storage account object (note the use of `range` here).  That should correspond to the name of the resource in the aspire manifest, which should be unique across all the names.